nemo_skills/dataset/dsbench_da/prepare.py

```
@@ -0,0 +1,274 @@
+        excel_paths = format_paths_for_prompt(
+            excel_files,
+            actual_root = extracted_data_dir,
+            display_root = display_root
+        )
+
+        # Get image files (for future multimodal support)
+        image_files = []
+        for ext in ['*.jpg', '*.png', '*.jpeg']:
+            image_files.extend(task_dir.glob(ext))
+
+        csv_files = list(task_dir.glob('*.csv'))
```

Ruff (0.15.1)

[error] Local variable `csv_files` is assigned to but never used

Remove assignment to unused variable `csv_files`

(F841)